projects/go-lib/src/lib/components/go-header-bar/go-header-bar.component.html

```diff
@@ -1,12 +1,22 @@
-<div class="go-header-bar" *ngIf="currentItem">
-  <a *ngIf="currentItem.showBackArrow"
-     (click)="processBackBtn()" class="go-header-bar__back-arrow">
-    <go-icon icon="arrow_back"></go-icon>
-  </a>
-  <h2 class="go-heading-2 go-header-bar__title">{{ currentItem.title }}</h2>
+<div
+  class="go-header-bar"
+  *ngIf="currentItem">
+
+  <go-icon
+    *ngIf="currentItem.showBackArrow"
+    (click)="processBackBtn()"
+    class="go-header-bar__back-arrow"
+    icon="arrow_back">
+  </go-icon>
+
+  <h2
+    class="go-heading-2 go-header-bar__title"
+    [ngClass]="{ 'go-header-bar__title--no-arrow': !currentItem.showBackArrow }">
+    {{ currentItem.title }}
+  </h2>
 
   <div class="go-header-bar__actions">
-    <ng-content select="[go-header-bar-action]"></ng-content>
```

Comment:
Ya I agree that this is not needed. We might want to add a deprecation warning on this PR just in case someone was using it, but it's not in the docs, so it shouldn't be used anywhere.